.github/workflows/testing.yml

```diff
@@ -0,0 +1,42 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        flake8 .
+    - name: Lint with black
+      run: |
+        pip install black
+        black --check .
+    - name: Install CGAL 5
+      run: |
+        sudo apt install -y libcgal-dev
+    - name: Install other dependencies
+      run: |
+        sudo apt-get install -y python3-pip
+        sudo apt install -y libopenmpi3 libopenmpi-dev openmpi-bin
+        sudo apt install -y libhdf5-dev
+        pip install .[io] --user
```

Comment:
No need to install pip or seismicmesh -- tox will handle it all.

Reply:
thanks, that's right. done.